python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -74,6 +74,7 @@ def evaluate(
         logs_dir: Optional[str] = None,
         logging_frequency: int = 25,
         compute_intermediate_stats: bool = True,
+        rank_distribution_t_test_pvalue_threshold: float = 0.1,
```

Comment:
Please add it to the docstring

Reply:
Since you touch the docstring: 
"Evaluate the RelevanceModel" can become something more explicit!

Reply:
ping